.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+. "$(dirname $0)/_/husky.sh"
+
+npm run lint
```

Comment:
```suggestion
npm run lint

```

Reply:
This adds newline at EOF

Reply:
I wonder if there's a way we can automate this if we need to add new hooks? 🤔 This is an auto generated file